src/main/java/uk/gov/di/ipv/service/ConfigurationService.java

```diff
@@ -22,7 +24,7 @@ public ConfigurationService(SSMProvider ssmProvider) {
     }
 
     public ConfigurationService() {
-        if (Boolean.parseBoolean(System.getenv("IS_LOCAL"))) {
+        if (Boolean.parseBoolean(System.getenv(IS_LOCAL))) {
```

Comment:
Would it be worth just calling the method on this class?
```suggestion
        if (isRunningLocally) {
```

Reply:
I did think that, but the method is an instance method, not static. I tried making it static but there were "problems".